drivers/remote/messages.go

```diff
@@ -93,6 +105,32 @@ func (r *createEndpointResponse) parseInterfaces() ([]*iface, error) {
 	return ifaces, nil
 }
 
+func (r *joinResponse) parseStaticRoutes() ([]*types.StaticRoute, error) {
+	var (
+		routes = make([]*types.StaticRoute, len(r.StaticRoutes))
+	)
+	for i, inRoute := range r.StaticRoutes {
+		var err error
```

Comment:
I don't think I can since err is used in a multiple assignment that also assigns a variable that was declared at a different scope.